examples/UI.Note-sample.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
+<edmx:Edmx Version="4.0" xmlns:edmx="http://docs.oasis-open.org/odata/ns/edmx">
+  <edmx:Reference Uri="/sap/opu/odata4/sap/ui_nte_demo_m_o4/srvd/sap/ui_nte_demo_m/0001/$metadata">
+    <edmx:Include Namespace="com.sap.gateway.srvd.ui_nte_demo_m.v0001" Alias="service" />
+  </edmx:Reference>
+  <edmx:Reference Uri="https://oasis-tcs.github.io/odata-vocabularies/vocabularies/Org.OData.Core.V1.xml">
+    <edmx:Include Namespace="Org.OData.Core.V1" Alias="Core" />
+  </edmx:Reference>
+  <edmx:Reference Uri="https://sap.github.io/odata-vocabularies/vocabularies/UI.xml">
+    <edmx:Include Namespace="com.sap.vocabularies.UI.v1" Alias="UI" />
+  </edmx:Reference>
+  <edmx:Reference Uri="https://sap.github.io/odata-vocabularies/vocabularies/Common.xml">
+    <edmx:Include Namespace="com.sap.vocabularies.Common.v1" Alias="Common" />
+  </edmx:Reference>
+  <edmx:DataServices>
+    <Schema Namespace="UI.examples" xmlns="http://docs.oasis-open.org/odata/ns/edm" Alias="self">
+      <Annotations Target="service.C_NTE_DEMO_MType">
+        <Annotation Term="UI.Facets">
+          <Collection>
+            <Record Type="UI.ReferenceFacet">
+              <PropertyValue Property="Target" AnnotationPath="_NoteBasic/@UI.Note" />
+            </Record>
+          </Collection>
+        </Annotation>
+      </Annotations>
+      <Annotations Target="service.C_NoteBasicTPType">
+        <Annotation Term="UI.Note">
+          <Record>
+            <PropertyValue Property="Title" Path="NoteBasicTitle" />
+            <PropertyValue Property="Content" Path="NoteBasicContent" />
+            <PropertyValue Property="Type" Path="NoteBasicType" />
+            <PropertyValue Property="MaximalLength" Path="_NoteBasicType/NoteBasicMaximalLength" />
+            <PropertyValue Property="MultipleNotes" Path="_NoteBasicType/NoteBasicMultipleNotes" />
+          </Record>
+        </Annotation>
+        <Annotation Term="Common.SAPObjectNodeType">
+          <Record>
+            <PropertyValue Property="Name" Path="_NoteBasicType/NoteBasicObjectNodeType" />
+          </Record>
+        </Annotation>
+        <Annotation Term="Common.CreatedBy" Path="CreatedByUser" />
+        <Annotation Term="Common.CreatedAt" Path="CreationDateTime" />
+        <Annotation Term="Common.ChangedBy" Path="LastChangedByUser" />
+        <Annotation Term="Common.ChangedAt" Path="LastChangeDateTime" />
+      </Annotations>
+      <Annotations Target="service.C_NoteBasicTPType/NoteBasicTitle">
```

Comment:
```suggestion
      <Annotations Target="service.C_NTE_DEMO_MType/_NoteBasic">
        <Annotation Term="Validation.MaxItems">
          <If>
            <Path>_NoteBasic/_NoteBasicType/NoteBasicMultipleNotes</Path>
            <Int>9223372036854775807</Int>
            <Int>1</Int>
          </If>
        </Annotation>
      </Annotations>
      <Annotations Target="service.C_NoteBasicTPType/NoteBasicTitle">
```
This would allow us to omit `@UI.Note/MultipleNotes`.